airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/__init__.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 from .assets import ENABLED_CHECKS as ASSETS_CHECKS
-from .documentation import ENABLED_CHECKS as DOCUMENTATION_CHECKS
+from .documentation.documentation import ENABLED_CHECKS as DOCUMENTATION_CHECKS
```

Comment:
💅 : You could declare `ENABLED_CHECKS` in the `documentation/__init__.py` file to avoid this change.

Reply:
updated